src/main/java/com/artipie/rpm/meta/XmlFile.java

```diff
@@ -157,4 +147,83 @@ private static XMLStreamWriter xmlStreamWriter(final Path path) {
             throw new UncheckedIOException("Failed to open file stream", err);
         }
     }
+
+    @Override
+    public void writeStartElement(
```

Comment:
@ammaratef45 these method should be part of `XmlWriterWrap`. The envelope class should take `XMLStreamWriter` in ctor